extensions/wikia/AdEngine/js/template/playwire.js

```diff
@@ -1,6 +1,6 @@
 /*global define*/
 define('ext.wikia.adEngine.template.playwire', [
-	'ext.wikia.adEngine.video.playwire',
+	'ext.wikia.adEngine.video.player.playwire.playwire',
```

Comment:
:see_no_evil: It hurts my eyes but I think I understand it: first playwire is the player's make and the second one is the player's name. Correct? :japanese_ogre: